dll/cpl/desk/lang/en-US.rc

```diff
@@ -184,9 +184,9 @@ BEGIN
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Compatibility", -1, 7, 67, 237, 97
     LTEXT "Some programs operate improperly if you do not restart your computer after you change color settings.\n\nAfter I change color settings", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&Restart the computer before applying the new color settings.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "Apply the new color settings without restarting.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Ask me before applying the new color settings.", IDC_ASKME_RB, 14, 146, 228, 10
+    AUTORADIOBUTTON "&Restart the computer before applying the new color settings", IDC_RESTART_RB, 14, 116, 228, 10
+    AUTORADIOBUTTON "Apply the new color settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10//FIXME: add accel
+    AUTORADIOBUTTON "Ask me before applying the new color settings", IDC_ASKME_RB, 14, 146, 228, 10//FIXME: add accel
```

Comment:
```suggestion
    AUTORADIOBUTTON "&Restart the computer before applying the new color settings", IDC_RESTART_RB, 14, 116, 228, 10
    AUTORADIOBUTTON "Apply the &new color settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
    AUTORADIOBUTTON "As&k me before applying the new color settings", IDC_ASKME_RB, 14, 146, 228, 10
```

Reply:
Please prove with a picture from en-US 2k3sp2 your suggestion that it is really &new and not a &p!!!!! Thank you in advance!

Reply:
Trust but verify I see 😂

This is XP SP2 but the resource is the same on 2003
![XP](https://github.com/user-attachments/assets/44e33a23-aa86-455f-aee3-ca602a9382ef)

Reply:
Here we are for w2k3sp2.

![W2K3SP2_advmon](https://github.com/user-attachments/assets/0a3e83bf-2125-4ff1-9d29-293f2f3ff567)

Reply:
Note that "color settings" (in our translations here) is wrong and should be "display settings".

Reply:
Thank you Doug for the en-US screenshot. I will apply those to the languages, which are known to follow en-US accels strictly.

Reply:
Resolved, according to the screenshot with &R &N &K for the en-US-alike languages for the 3ARBs.